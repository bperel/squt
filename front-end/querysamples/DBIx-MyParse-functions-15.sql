SELECT CAST(expr AS SIGNED)
