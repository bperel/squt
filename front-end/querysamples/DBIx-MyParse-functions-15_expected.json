{
  "Warning": {
    "No alias": {
      "cast_as_signed": "SELECT"
    }
  },
  "Tables": {
    "": {
      "": {
        "OUTPUT": {
          "expr": {
            "0": "expr"
          }
        }
      }
    }
  },
  "Functions": {
    "0": {
      "to": "OUTPUT",
      "name": "cast_as_signed",
      "alias": "0"
    }
  }
}
